pipelines/_ingredients/README.md

```diff
@@ -0,0 +1,8 @@
+# Pipeline Definition Ingredients
+
+This directory contains ingredient pipeline definition YAML files.
+
+The pipelines defined here are intended to be used as building blocks for more complex pipelines.
+As such, they are not intended to be used directly by end-users for science purposes.
+
+Science ready pipeline definitions are located in the [pipelines](..) directory.
```

Comment:
```suggestion
The pipelines defined here are intended to be used as building blocks for more complex pipelines.
As such, they are not intended to be used directly by end-users for verification purposes.

Standalone pipeline definitions are located in the [pipelines](..) directory.
```

Reply:
Modifying this sentence to end with "by end-users.", as discussed above.

Reply:
Replacing "Standalone" with "User-facing". The term standalone to me implies that these may be used in isolation, but more often than not they import a series of other pipeline YAMLs in order to function correctly.